### ISSUE TITLE
about FLUX version for training subject-driven model

### ISSUE THREAD

**Issue by Weizhi-Zhong** [Yuanshi9815/OminiControl#59]
Hi, thanks for your open source. 
I have some question about the flux version.
I find that the flux_path in train/config/subject_512.yaml is "black-forest-labs/FLUX.1-dev".
But as said in Limitations, the subject-driven generation model may not work well with FLUX.1-dev.
Which version should we use for training subject-driven model.

The results of training with FLUX.1-dev after 15,000 iterations indeed seem inferior to the results of  "experimental/subject" model.

**Comment by Yuanshi9815** [comment]
Hi @Weizhi-Zhong ,

Thanks for raising this question. 

The model performance should be better than the model `experimental/subject` since the training settings in the released code are more optimized. If you're seeing inferior results, here are some suggestions:

1. Try increasing `gradient accumulation steps` in your yaml config for more stable training.
2. Experiment with different checkpoints (e.g. 14k, 13k, or even 5k iterations) as training stability can vary.

The training process isn't completely stable in our observations, so trying these adjustments may help improve your results. 
Additionally, we're optimizing our training code and will continue to release improved training settings through updates.
